FreeAPS/Sources/Services/WatchManager/WatchManager.swift

```diff
@@ -288,6 +296,30 @@ extension BaseWatchManager: WCSessionDelegate {
             }
         }
 
+        if let nutrientsData = message["addNutrients"] as? Data {
+            if let nutrients = try? JSONDecoder().decode([Nutrient: Int].self, from: nutrientsData) {
+                if nutrients.values.contains(where: { $0 > 0 }) {
+                    addCarbsStateModel.carbs = Decimal(nutrients[.carb] ?? 0)
+                    addCarbsStateModel.protein = Decimal(nutrients[.protein] ?? 0)
+                    addCarbsStateModel.fat = Decimal(nutrients[.fat] ?? 0)
+                    addCarbsStateModel.add()
```

Comment:
🚨 The app is crashing on this step

Reply:
<img width="1323" alt="Screenshot 2023-07-05 at 16 36 21" src="https://github.com/Artificial-Pancreas/iAPS/assets/17122466/aa1ebb5c-9f1c-4a71-bade-69df31fc8e04">

Reply:
Fixed